src/Languages/Python.ts

```diff
@@ -396,7 +398,7 @@ export class Python extends PythonicLanguage {
             "min",
             NativeCallScope.Static,
             NativeCallType.Function);
-        math.requiredImports = [];
+        math.requiredImports = [new Import(["math"], ["fabs", "ceil", "floor"], ImportRelativity.Absolute)];
```

Comment:
If a file just calls, say, `math ceil`, it looks weird to have it also import `fabs`, `floor`, and any other math commands that get added later. They should either import just what they need or overall import `math`.

Also, since you seem to be a Python person 😄, is there some sort of standard for when to use which form? I'm wondering if it makes sense to add a way to have imports "collapse" into the full form if there are too many imported items.